h/api/test/models_test.py

```diff
@@ -462,13 +462,13 @@ def test_document_link_truncates_hostname(hostname_or_filename, href, title,
     link_text.return_value = "www.example.com/example.html"
 
     # .hostname_or_filename is too long.
-    hostname_or_filename.return_value = "a" * 60
+    hostname_or_filename.return_value = "a" * 70
 
-    expected_hostname = "a" * 50 + "&hellip;"
+    expected_hostname = "a" * 60 + "&hellip;"
     expected_result = (
         '<a href="http://www.example.com/example.html" '
         'title="www.example.com/example.html">'
-        'www.example.com/example.html</a> ({hostname})'.format(
+        'www.example.com/example.html</a><br/>({hostname})'.format(
```

Comment:
Could you replace these with plain old `<br>`? Self-closing tags aren't a thing in HTML5.